about/index.md

```diff
@@ -14,7 +14,7 @@ Keyboard layouts are distributed through an open catalog to all major desktop an
 ## About SIL Global
 
 <!--div style=style='float:left; margin: 0 16px 0 0'-->
-![](/cdn/dev/img/sil-logo-blue-2017_1.png)
+![](/cdn/dev/img/sil-logo-abbysinica-293-159.png)
```

Comment:
These paths will now point to `/ _common/assets/...` from keymanapp/shared-sites#21

Reply:
Hi @darcywong00. 
May I keep the current path? The logo from _common/assets/ is big
![image](https://github.com/user-attachments/assets/030c34e7-26bb-4610-b8df-d25598d0199b)

Reply:
`![](/cdn/dev/img/sil-logo-abbysinica-293-159.png)`
![image](https://github.com/user-attachments/assets/a20e9aaa-909d-4dd3-a0a6-25235cd7ec20)

Reply:
My concern is having multiple copies of the logo making it hard to maintain.

I saw the footer is using `width=50%`. Maybe we just use HTML in the Markdown
```html
<img id="sil-logo" src="/_common/assets/sil-logos-2024/sil-logo-abbysinica.png" width="50%" alt='SIL' />
```